emails/models.py

```diff
@@ -157,6 +157,11 @@ def emails_blocked(self):
         ).values('num_blocked')
         return sum(blocked['num_blocked'] for blocked in relay_addresses_blocked)
 
+    @property
+    def created_before_premium_release(self):
+        date_created = self.user.date_joined
+        return date_created < datetime(2021, 10, 27, 18, 00) 
```

Comment:
Can we make this come from an env var so it's not hard-coded? Like, `2021-10-27` ?